# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/sysvinit/dev_vc_0.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur sysvinit-2.85.orig/src/bootlogd.c sysvinit-2.85/src/bootlogd.c
--- sysvinit-2.85.orig/src/bootlogd.c	2001-12-09 15:01:28.000000000 +0100
+++ sysvinit-2.85/src/bootlogd.c	2003-08-30 15:49:12.000000000 +0200
@@ -318,7 +318,7 @@
 	(void)ioctl(0, TIOCCONS, NULL);
 #if 1
 	/* Work around bug in 2.1/2.2 kernels. Fixed in 2.2.13 and 2.3.18 */
-	if ((n = open("/dev/tty0", O_RDWR)) >= 0) {
+	if ((n = open("/dev/vc/0", O_RDWR)) >= 0) {
 		(void)ioctl(n, TIOCCONS, NULL);
 		close(n);
 	}
diff -Nur sysvinit-2.85.orig/src/paths.h sysvinit-2.85/src/paths.h
--- sysvinit-2.85.orig/src/paths.h	2003-04-14 13:37:01.000000000 +0200
+++ sysvinit-2.85/src/paths.h	2003-08-30 15:49:38.000000000 +0200
@@ -13,7 +13,7 @@
  *		as published by the Free Software Foundation; either version
  *		2 of the License, or (at your option) any later version.
  */
-#define VT_MASTER	"/dev/tty0"		/* Virtual console master */
+#define VT_MASTER	"/dev/vc/0"		/* Virtual console master */
 #define CONSOLE		"/dev/console"		/* Logical system console */
 #define SECURETTY	"/etc/securetty"	/* List of root terminals */
 #define SDALLOW		"/etc/shutdown.allow"	/* Users allowed to shutdown */
